src/frontend/screens/PhotosModal/index.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect, useMemo, useRef, useState } from "react";
+import { Dimensions, View, StyleSheet } from "react-native";
+import { NavigationState, TabView, Route } from "react-native-tab-view";
+
+import IconButton from "../../sharedComponents/IconButton";
+import Button from "../../sharedComponents/Button";
+import { CloseIcon, DeleteIcon } from "../../sharedComponents/icons";
+import { filterPhotosFromAttachments } from "../../lib/utils";
+import { useObservation } from "../../hooks/useObservation";
+import { useDraftObservation } from "../../hooks/useDraftObservation";
+import PhotoView from "../../sharedComponents/PhotoView";
+import api from "../../api";
+import type { NavigationProp } from "../../types";
+import { useBottomSheetModal } from "../../sharedComponents/BottomSheetModal";
+import { ConfirmDeleteModal } from "./ConfirmDeleteModal";
+import Text from "../../sharedComponents/Text";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { TouchableNativeFeedback } from "../../sharedComponents/Touchables/index.ios";
+
+interface customRoute extends Route {
+  uri?: string;
+  error?: boolean;
+  capturing?: boolean;
+}
+
+const m = defineMessages({
+  deleteImage: {
```

Comment:
should a description be included here? idk if we settled on a convention moving forward to include whenever possible or not

Reply:
Description are only necessary when the context of the translation is ambiguous. For example, and words or phrases that have more then one meaning